lang/typescript/src/extended-address/splitAddress1.test.ts

```diff
@@ -3,34 +3,111 @@ import {splitAddress1} from './splitAddress1';
 describe('splitAddress1', () => {
   test('returns null when extended address is not defined for region', () => {
     expect(splitAddress1('US', '123 Main')).toBeNull();
+    expect(splitAddress1('US', '123 Main', true)).toBeNull();
```

Comment:
The current behavior is to return null if the country has no region config, regardless of what is inputted. Here's some new test cases that'll cover that.

```suggestion
    expect(splitAddress1('US', '123 Main')).toBeNull();
    expect(splitAddress1('US', '')).toBeNull();
    expect(splitAddress1('US', '123 Main', true)).toBeNull();
    expect(splitAddress1('US', '', true)).toBeNull();
```